cpp/libcudf_kafka/src/kafka_consumer.cpp

```diff
@@ -173,8 +193,6 @@ std::map<std::string, int64_t> kafka_consumer::get_watermark_offset(std::string
     if (err == RdKafka::ErrorCode::ERR__PARTITION_EOF) {
       results.insert(std::pair<std::string, int64_t>("low", low));
       results.insert(std::pair<std::string, int64_t>("high", high));
-    } else {
-      CUDF_FAIL("Error retrieving Kafka watermark offset from broker");
```

Comment:
Is this related to the other changes?

Reply:
@cwharris Thank you for catching this. This is a mistake and part of something else I am working on but seems to have mistakenly found its way in here. I will make another commit to replace it.